src/WebJobs.Extensions.ApiHub/ApiHubFileAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.ApiHub.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
+{
+    public class ApiHubFileAttribute : Attribute, IFileAttribute
```

Comment:
Add doc comments throughout. Fxcop should be complaining - perhaps you turned it off temporarily? If so, please run.